orchestrator/careplancontributor/service.go

```diff
@@ -117,6 +145,112 @@ func (s Service) handleProxyToEPD(writer http.ResponseWriter, request *http.Requ
 	proxy.ServeHTTP(writer, request)
 }
 
+func (s Service) handleProxyToFHIR(writer http.ResponseWriter, request *http.Request) error {
+	// Authorize requester before proxying FHIR request
+	// Data holder must verify that the requester is part of the CareTeam by checking the URA
+	// Validate by retrieving the CarePlan from CPS, use URA in provided token to validate against CareTeam
+	// CarePlan should be provided in X-SCP-Context header
+	xScpContext := request.Header["X-Scp-Context"]
+	if len(xScpContext) == 0 {
+		// Check if header has different capitalisation, it may be changed by Go canonicalization
+		xScpContext = request.Header["X-SCP-Context"]
+		if len(xScpContext) == 0 {
+			err := errors.New("X-SCP-Context header value must be set")
+			http.Error(writer, err.Error(), http.StatusUnauthorized)
```

Comment:
I'd say only return the error, and convert to OperationOutcome where it's called

Reply:
Yeah that aligns with what I was thinking as well. This was good for initial tests but I agree it should change. Will do